diff --git a/node_modules/react-navigation-shared-element/lib/typescript/SharedElement.d.ts b/node_modules/react-navigation-shared-element/lib/typescript/SharedElement.d.ts
index 39d47a2..7441ac2 100644
--- a/node_modules/react-navigation-shared-element/lib/typescript/SharedElement.d.ts
+++ b/node_modules/react-navigation-shared-element/lib/typescript/SharedElement.d.ts
@@ -1,8 +1,9 @@
 import * as React from 'react';
+import { ViewProps } from 'react-native';
 export declare type SharedElementProps = {
     id: string;
     children: React.ReactNode;
-};
+} & ViewProps;
 declare class SharedElement extends React.Component<SharedElementProps> {
     private node;
     private sharedId;
