diff --git a/node_modules/react-native-hooks/lib/useKeyboard.js b/node_modules/react-native-hooks/lib/useKeyboard.js
index 1264896..70bb67a 100644
--- a/node_modules/react-native-hooks/lib/useKeyboard.js
+++ b/node_modules/react-native-hooks/lib/useKeyboard.js
@@ -2,14 +2,14 @@ import React, { useEffect, useState } from 'react'
 import { Keyboard } from 'react-native'
 
 export default function useKeyboard() {
-  const [keyboard, setKeyboard] = useState({ Keyboard })
+  const [keyboard, setKeyboard] = useState({ Keyboard, isKeyboardShow: false})
 
   function keyboardShown(e) {
     setKeyboard({
       isKeyboardShow: true,
       Keyboard,
-      endCoordinates: e.endCoordinates,
-      startCoordinates: e.startCoordinates
+      endCoordinates: e ? e.endCoordinates : null,
+      startCoordinates: e ? e.startCoordinates : null
     })
   }
 
@@ -17,8 +17,8 @@ export default function useKeyboard() {
     setKeyboard({
       isKeyboardShow: false,
       Keyboard,
-      endCoordinates: e.endCoordinates,
-      startCoordinates: e.startCoordinates
+      endCoordinates: e ? e.endCoordinates : null,
+      startCoordinates: e ? e.startCoordinates : null
     })
   }
 
